packages/libraries/yoga/src/index.ts

```diff
@@ -179,13 +179,10 @@ export function useHive(clientOrOptions: HiveClient | HivePluginOptions): Plugin
         ? clientOrOptions
         : createHive({
             ...clientOrOptions,
+            // TODO: How to convert yoga.logger into HiveLogger here?
+            logger: clientOrOptions.logger ?? yoga.logger,
```

Comment:
@enisdenjo Any ideas here? 😓 

Basically, if the user provides the logger -> we use that one, otherwise we "convert" the yoga logger and pass it in here? Right now the yoga.logger types are incompatible..